style.css

```diff
@@ -70,6 +96,14 @@
   padding: 11px 0px 0px 5px;
 }
 
+.header-container {
+  background-image: linear-gradient(to top, rgb(0, 0, 0, 0.3) 0%, transparent 1%), url("assets/header_fon.jpg");
+  background-size: 100%;
+  height: 566px;
+  width: 1300px;
```

Comment:
Use use `1300px;` in multiple places, i am pretty sure you can refactor this. If your container element has 1300 width it's children won't overflow